pkg/escape/bytes.go

```diff
@@ -10,6 +10,22 @@ func Bytes(in []byte) []byte {
 }
 
 func Unescape(in []byte) []byte {
+	if len(in) == 0 {
+		return nil
+	}
+
+	var hasEscape bool
+	for _, b := range in {
+		if b == '\\' {
```

Comment:
Does ranging over a byte slice and comparing perform better than bytes.IndexByte, which is apparently [implemented in assembly](https://golang.org/src/bytes/bytes_decl.go?s=287:323#L1)?  Same question for the string escape, with [strings.IndexByte](https://golang.org/src/strings/strings_decl.go?s=274:310#L1).